pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -339,4 +342,21 @@ public ConsumerConfiguration setProperties(Map<String, String> properties) {
     public ConsumerConfigurationData getConfigurationData() {
         return conf;
     }
+    
+     /** 
+     * @param subscriptionInitialPosition the initial position at which to set
+     * set cursor  when subscribing to the topic first time
+     * Default is {@value InitialPosition.Latest}
+     */
+    public ConsumerConfiguration setSubscriptionInitialPosition(InitialPosition subscriptionInitialPosition) {
```

Comment:
`ConsumerConfiguration` class has already been set as deprecated for 2.0 release. The new setting is only needed in the new `ConsumerBuilder` interface.